src/core/component.js

```diff
@@ -278,7 +278,7 @@ Component.prototype = {
 
     // Just cache the attribute if the entity has not loaded
     // Components are not initialized until the entity has loaded
```

Comment:
Perhaps update this comment to something like "... until the entity starts loading"